### ISSUE TITLE
인증완료시 콜백함수가 실행되지 않고 mRedirectUrl 을 요구하는데 어떻게 해야 될까요?

### ISSUE THREAD

**Issue by gamcho3** [portone-io/portone_flutter#74]

**Comment by anymate98** [comment]
이니시스 통합인증으로 테스트하고 계신가요?

**Comment by gamcho3** [comment]
> 이니시스 통합인증으로 테스트하고 계신가요?

네 그렇습니다. "정상적인 통합인증 처리를 위해 M_redirect_url 파라메터 세팅이 필요합니다." 라고 뜨고 pushReplacement가 되지 않네요.

**Comment by anymate98** [comment]
이니시스의 경우 현재 웹뷰 상에서는 리다이렉트 방식으로 동작하기 때문에 mRedirectUrl 파라미터가 필수입니다. http로 시작하는 url 아무거나 적어주시면 이용가능합니다.

**Comment by gamcho3** [comment]
감사합니다. 혹시 다날 결제서비스로 본인인증 테스트를 해보려면 어떻게 해야 할까요?

**Comment by anymate98** [comment]
예제에는 다날 본인인증이 설정되어 있으니 실제 실행과정을 보고싶으시다면 예제를 활용하시면 됩니다.
만약 실제 사용하실 수 있는 계정을 갖고 계시다면 그걸 사용할 수도 있습니다.

**Comment by gamcho3** [comment]
휴대폰 본인인증 예제로 테스트를 하고 있는데 이니시스가 떠서요 ㅠㅠ

**Comment by anymate98** [comment]
이 repo의 example 폴더의 앱을 실행해보시라는 뜻이었습니다.

**Comment by anymate98** [comment]
'iamport'에 이니시스 통합인증이 설정되어있는 것은 확인했습니다.

**Comment by gamcho3** [comment]
usercode를 변경하는거였군요. 친철한 설명 감사합니다 ㅎㅎ
